### PR TITLE
doc/mgr: edit "Overview" in dashboard.rst

### PR DESCRIPTION
**Pull request by zdover23**
Edit and update the "Overview" section in doc/mgr/dashboard.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -6,33 +6,29 @@ Ceph Dashboard
...
-The dashboard module's backend code uses the CherryPy framework and implements
-a custom REST API. The WebUI implementation is based on
-Angular/TypeScript and includes both functionality from the original dashboard
-and new features originally developed for the standalone version
-of openATTIC. The Ceph Dashboard module is implemented as an
-application that provides a graphical representation of information and statistics
-through a web server hosted by ``ceph-mgr``.
+The Ceph Dashboard is a built-in web-based Ceph management-and-monitoring
+application used to inspect and administer resources in the cluster. It is
+implemented as a :ref:`ceph-manager-daemon` module.
```
Comment (anthonyeleven) [inline doc/mgr/dashboard.rst:11]: Suggest:
>The Ceph Dashboard is a web-based management and monitoring tool that can be used to view cluster and component status and administer resources.

**Review by anthonyeleven — APPROVED** [review]

File: doc/mgr/dashboard.rst (hunk 1 above)
Comment (zdover23) [reply doc/mgr/dashboard.rst:11]: I like that better.

@anthonyeleven, don't merge this one until Ilya and Neha and Dan and Josh have been consulted and satisfied. I want to make sure to update the contributors to the Dashboard while respecting the historical contributions of SUSE. But I want to make sure that they are satisfied before I start ripping out vital organs.

Comment (anthonyeleven) [reply doc/mgr/dashboard.rst:11]: Ack, was already leaving the merge to you.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/57335 - Squid backport
https://github.com/ceph/ceph/pull/57336 - Reef backport
https://github.com/ceph/ceph/pull/57337 - Quincy backport

**Pull request merged**